spec/watir/rspec/matchers/base_matcher_spec.rb

```diff
@@ -61,17 +61,17 @@
         end
       end
 
-      object.should be_foo.within(1)
-      raised.should be_true
+      expect(object).to be_foo.within(1)
+      expect(raised).to be_truthy
```

Comment:
Please use be_true here since we care in this test that it is actually true and not anything else, because anything else might mean that functionality is not working correctly.